transmute_core/function/response.py

```diff
@@ -1,6 +1,10 @@
 import attr
-from collections import Mapping
+import sys
 
+if  sys.version_info.major == 3 and sys.version_info.minor >= 10:
+    from collections.abc import Mapping
+else:
+    from collections import Mapping
```

Comment:
it seems like collections.abc has had mapping since 3.3...

instead of looking for specific versions, how about doing a try/catch on import error instead, with collections.abc taking precedence? This'll grab Mapping from wherever it's supported.